docs/sources/configure-user-settings/_index.md

```diff
@@ -46,12 +46,33 @@ To learn more about RBAC for Grafana OnCall, refer to the following documentatio
 
 ## Manage Teams in Grafana OnCall
 
-Teams in Grafana OnCall are based on the teams created at the organization level of your Grafana instance,
-in **Configuration > Teams**. Administrators can create a different configuration for each team, and can navigate
-between team configurations in the **Select Team** dropdown menu in the **Alert Group** section of Grafana OnCall.
-
-Users, including admins, can only view and manage teams in OnCall if they are a member of that team.
-An admin user may need to temporarily add themselves to a team to manage it.
+Teams in Grafana OnCall enable the configuration of visibility and filtering of resources, such as alert groups,
+integrations, escalation chains, and schedules. OnCall teams are automatically synced with
+[Grafana teams](https://grafana.com/docs/grafana/latest/administration/team-management/) created at the organization
+level of your Grafana instance. To modify global settings like team name or team members, navigate to
+**Configuration > Teams**. For OnCall-specific team settings,
+go to **Alerts and Incidents > OnCall > Settings > Teams and Access Settings**.
+
+This section displays a list of teams, allowing you to configure team visibility and access to team resources for all
+Grafana users, or only admins and team members. You can also set a default team, which is a user-specific setting;
+the default team will be pre-selected each time a user creates a new resource. The team list includes a `No team` tag,
+signifying that the resource has no team and is accessible to everyone.
+
+Admins can view the list of all teams, while editors and viewers can only see teams (and their resources)
```

Comment:
probably not something to address now, but w/ the introduction of RBAC, we should start to consider how we document references to the basic roles.